news_hounds_django_server/urls.py

```diff
@@ -1,12 +1,14 @@
+from rareapi.views.PostTags import PostTagsViewSet
```

Comment:
This import seems unnecessary since we are importing this with the other ViewSets on line 5, right? Should we get rid of this?

Reply:
think that was an autoimport, will ditch